suite-native/module-connect-popup/src/hooks/useConnectParseParams.ts

```diff
@@ -3,10 +3,31 @@ import { useMemo } from 'react';
 import { ParsedURL } from 'expo-linking';
 
 import TrezorConnect from '@trezor/connect';
+import { DEEPLINK_VERSION } from '@trezor/connect/src/data/version';
 
-export const useConnectParseParams = (url: ParsedURL) => {
-    const popupOptions = useMemo(() => {
-        const { queryParams } = url;
+export enum ParseParamsErrors {
+    INVALID_PARAMS = 'invalid_params',
```

Comment:
We probably could user here 

```typescript
throw ERRORS.TypedError( 'Method_InvalidParameter');
```

Reply:
True but I can't throw here in `useMemo`

Reply:
Sure, but maybe just using `ERRORS.TypedError( 'Method_InvalidParameter')` ?

Reply:
Yes, that looks awesome.